translations/plurals.ts

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="en">
```

Comment:
Ok.

I guess it was changed because I loaded it into QtLinguist. I fell back to 2.0 in the pl translation too.